src/eatery_db/common_eatery.py

```diff
@@ -1,4 +1,38 @@
 from constants import IMAGES_URL
+from datetime import date, datetime, timedelta
+from database import ExpandedMenuStation, ExpandedMenuItem, ExpandedMenuChoice
+
+today = date.today()
```

Comment:
I don't think this should be calculated at the top. It might only get initialized when the server starts but never gets recalculated again. This line should prob be in the functions that use it that way when the function gets called, this variable is recalculated. This might be a possible cause to some of our issues